src/PIL/PcfFontFile.py

```diff
@@ -207,7 +210,7 @@ def _load_bitmaps(self, metrics):
 
         data = fp.read(bitmapsize)
 
-        pad = BYTES_PER_ROW[padindex]
+        pad: Callable[[int], int] = BYTES_PER_ROW[padindex]
```

Comment:
I don't see why this one is needed?

Checking with `typing.reveal_type`, I see that it can be inferred if you add the following at the top of the file:
```python
BYTES_PER_ROW: list[Callable[[int], int]] = [ ...
```

Reply:
Without it, `python3 -m mypy --strict src/PIL/PcfFontFile.py` gives
```
src/PIL/PcfFontFile.py:223: error: Call to untyped function (unknown) in typed context  [no-untyped-call]
                    Image.frombytes("1", (xsize, ysize), data[b:e], "raw", mode, pad(xsize))
                                                                                 ^~~~~~~~~~
Found 1 error in 1 file (checked 1 source file)
```

Reply:
Your suggestion also works however, so I've pushed a commit.

Reply:
Ah, I was missing the `--strict`.